h5io/_h5io.py

```diff
@@ -292,7 +293,7 @@ def _triage_read(node, slash='ignore'):
             from pandas import read_hdf
             rootname = node.name
             filename = node.file.filename
-            data = read_hdf(filename, rootname, mode='r')
+            data = read_hdf(filename, rootname)
```

Comment:
it's actually unclear to me exactly what changed...it has a **kwargs section so I think that may have been passed to a subsequent function, which itself stopped supporting `mode`. So it could be that I wasn't using it in a totally standard kind of way.